### PR TITLE
Add torsf test (pluggable transport)

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Related to https://github.com/ooni/ooni.org/issues/771 https://github.com/ooni/ooni.org/issues/772
Implements https://github.com/ooni/ooni.org/issues/766

### CODE REVIEW

**Review by bassosimone — APPROVED** [review]
🐳

File: af/fastpath/fastpath/core.py [hunk 1]
```diff
@@ -1251,6 +1251,22 @@ def score_signal(msm) -> dict:
...
+def score_torsf(msm) -> dict:
+    """Calculate measurement scoring for Tor Snowflake
+    Returns a scores dict
+    """
+    # https://github.com/ooni/ooni.org/issues/772
+    scores = init_scores()
+    tk = msm.get("test_keys", {})
+    failure = tk.get("failure")
+    if failure:
+        scores["accuracy"] = 0.0
```
Comment (hellais) [inline af/fastpath/fastpath/core.py]: I don't think you should be setting the accuracy to zero in the case of `failure`. The meaning of the `failure` key is to indicate that there is an instance of blocking of the protocol.

**Comment by FedericoCeratto** [comment]
"torsf" test name added to database

File: af/fastpath/fastpath/core.py (hunk 1 above)
Comment (FedericoCeratto) [reply af/fastpath/fastpath/core.py]: Fixed

**Pull request merged**